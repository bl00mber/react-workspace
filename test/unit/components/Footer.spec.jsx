import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Footer from '@/components/Footer';

describe('Footer', () => {
  test('should match enzyme shallow snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
