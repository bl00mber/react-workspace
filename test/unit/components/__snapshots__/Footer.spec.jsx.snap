// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match enzyme shallow snapshot 1`] = `
<p>
  Show:
   
  <Connect(Link)
    filter="SHOW_ALL"
  >
    All
  </Connect(Link)>
  , 
  <Connect(Link)
    filter="SHOW_ACTIVE"
  >
    Active
  </Connect(Link)>
  , 
  <Connect(Link)
    filter="SHOW_COMPLETED"
  >
    Completed
  </Connect(Link)>
</p>
`;
